### ISSUE TITLE
System.UnauthorizedAccessException when running versioninfo from C:\

### ISSUE THREAD

**Issue by devinvisible** [taylorjg/dotnet-versioninfo#1]
When running under an Administrator account in an administrator command prompt, I'm seeing `System.UnauthorizedAccessException` being thrown. This scenario will surely happen so it just needs to be handled more gracefully (perhaps even a flag to to optionally hide [even clean looking] error messages).
```
C:\>versioninfo **/test*.dll
```
```
Error finding file system entries in C:\ProgramData\Application Data: System.UnauthorizedAccessException: Access to the path 'C:\ProgramData\Application Data' is denied.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path)
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerableFactory.FileSystemInfos(String directory, String expression, EnumerationOptions options)
   at System.IO.DirectoryInfo.InternalEnumerateInfos(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.DirectoryInfo.EnumerateFileSystemInfos()
   at System.IO.Abstractions.DirectoryInfoWrapper.EnumerateFileSystemInfos()
   at Ganss.IO.Glob.Expand(String path, Boolean dirOnly)+MoveNext().
```

Piggy backing on a similar complaint, I also noticed the following with the same call stack:
```
Error finding file system entries in C:\$Recycle.Bin\S-1-5-21-89625216-2347251461-3949967815-1000\$RWUDRHQ\ProcessInjection\CurrentVersion: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\$Recycle.Bin\S-1-5-21-89625216-2347251461-3949967815-1000\$RWUDRHQ\ProcessInjection\CurrentVersion'.
```

**Comment by taylorjg** [comment]
I am using [Glob.cs](https://www.nuget.org/packages/Glob.cs) for globbing. I currently set the `ErrorLog` property on the `Glob` object to `Console.WriteLine`. This approach should write errors to the console but continue processing. However, the error message passed to the `ErrorLog` action seems to include the whole stack trace too.

Instead, I can set `ThrowOnError` to `true` in which case, I have control over the format of the error message. However, processing will presumably stop at that point which is not desirable.

If `ErrorLog` was called like this:

```
Log("Error finding file system entries in {0}: {1}.", parentDir, ex.Message);
```

instead of this:

```
Log("Error finding file system entries in {0}: {1}.", parentDir, ex);
```

we would be fine.

I suppose I could open a PR in the [Glob.cs](https://github.com/mganss/Glob.cs) repo.

**Comment by taylorjg** [comment]
_it seems this issue was automatically closed when I pushed a commit referencing this issue_

I have made a change whereby I remove the stack trace from error messages passed to the `ErrorLog` action. I released this change as `1.0.2`. Perhaps you can give it a try.

**Comment by devinvisible** [comment]
I tried `1.0.2` and the call stacks are no longer written. Instead I simply see the error message.

I would really love to see an additional flag which would suppress logging errors in entirety. I'm an administrator in an administrator command prompt and there are directories that will always throw UnauthorizedAccessException (and potentially DirectoryNotFoundException). This is okay but I don't want it cluttering the output.
